app/src/main/java/org/sagebionetworks/research/mpower/tracking/widget/SymptomsLoggingUIFormItemWidget.java

```diff
@@ -0,0 +1,94 @@
+package org.sagebionetworks.research.mpower.tracking.widget;
+
+import android.content.Context;
+import android.graphics.Paint;
+import android.support.annotation.NonNull;
+import android.support.constraint.ConstraintLayout;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.widget.RadioButton;
+import android.widget.TextView;
+
+import org.sagebionetworks.research.mobile_ui.widget.ActionButton;
+import org.sagebionetworks.research.mpower.MPowerRadioButton;
+import org.sagebionetworks.research.mpower.R;
+import org.sagebionetworks.research.mpower.tracking.fragment.SymptomLoggingFragment;
+
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.BindViews;
+import butterknife.ButterKnife;
+
+public class SymptomsLoggingUIFormItemWidget extends ConstraintLayout {
```

Comment:
All of these views don't conform to any color theme, is that something you're planning to do on Android to match iOS?

Reply:
Color themes are also not implemented on the android side yet. Eventually that is the goal hopefully we will have time to do this and color styles soon.